src/realm/array_direct.hpp

```diff
@@ -195,7 +195,7 @@ class unaligned_word_iter {
     }
     // 'num_bits' number of bits which must be read
     // WARNING returned word may be garbage above the first 'num_bits' bits.
-    uint64_t get(unsigned num_bits)
+    uint64_t get(uint64_t num_bits)
```

Comment:
num_bits will never be large, so this change just makes the code slower on 32 bit platforms.